src/api/wix-api-context-provider.tsx

```diff
@@ -106,7 +113,61 @@ function getWixApi(wixClient: ReturnType<typeof getWixClient>) {
             // return { success: true, url: redirectSession?.fullUrl };
             return { success: true, url: '' };
         },
-        getMyProfile: async () => await wixClient.members.getCurrentMember(),
+        getMyProfile: async () => await wixClient.members.getCurrentMember({}),
+        initiateLogin: async () => {
+            const loginRequestData = wixClient.auth.generateOAuthData(
+                "http://localhost:5173",
```

Comment:
you should send your current domain

Reply:
this one still looks relevant.

Reply:
this wont work as the URL needs to be saved in the wix backoffice so the user will need to add his own url's and only then the "window.location.origin" will work